src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -18,11 +18,14 @@ package nl.knaw.dans.easy
 import java.net.{HttpURLConnection, URL}
 
 import nl.knaw.dans.pf.language.ddm.handlermaps.NameSpace
-import nl.knaw.dans.pf.language.emd.types.EmdConstants
 
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
 package object stage {
+
+  // exceptions
+  case class RejectedDepositException(msg: String = "", cause: Throwable = null) extends Exception(msg, cause)
```

Comment:
The idea of logging the error right away was in combination of using Either in place of Try.  Now the idea is indeed to propagate also the details upstairs, and do logging in one central place.